gradle.properties

```diff
@@ -17,3 +17,5 @@
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
 android.useAndroidX=true
+
+VERSION_NAME=0.9.2
```

Comment:
I'd rather change this in only one place instead of each module's build file. :sweat_smile: